src/app/components/form/form.component.html

```diff
@@ -79,13 +79,13 @@ <h1>{{'Domain name'|translate}}</h1>
                 <div class="custom-control custom-checkbox custom-control-inline">
                   <input type="checkbox" class="custom-control-input" formControlName="disable_ipv4" id="disable_protocol_ipv4" [class.is-invalid]="newForm.errors?.noProtocol || newForm.controls.disable_ipv4.invalid">
                   <label class="custom-control-label" for="disable_protocol_ipv4">
-                    {{'Disable IPv4' |translate}}
+                    {{ 'DISABLE_IP_CHECKBOX' | translate: {ip: 'IPv4'} }}
```

Comment:
Since you have explicitly defined the string, I think you can keep the full string instead of using the variable I introduced in https://github.com/zonemaster/zonemaster-gui/pull/243/commits/710f06ef2cb7ac501b6564d9ed6cf25f828394af.
(it would imply that each LANG.json should be updated with the full string as added in #218)